### ISSUE TITLE
```make install``` rule fails

### ISSUE THREAD

**Issue by miquelramirez** [miccol/Behavior-Tree#13]
After completing the build, the command

```
sudo make install
```

fails with the following output

```
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/lib/BTpp/libBTpp.a
-- Installing: /usr/local/lib/BTpp/btpp-targets.cmake
-- Installing: /usr/local/lib/BTpp/btpp-targets-release.cmake
CMake Error at cmake_install.cmake:59 (file):
  file INSTALL cannot find
  "/home/bowman/Sandboxes/libbtpp/btpp-config.cmake".


Makefile:61: recipe for target 'install' failed
```

Looks like the support for ```pkg-config``` is not complete in the ```master``` branch?

PS: Also, the installation rules for headers seem to be broken/incomplete on ```master```.

**Comment by miccol** [comment]
I will have a look into that. Thank you!

**Comment by miccol** [comment]
I did some quick changes in the `installation-rules` branch.

I will test it with all the OSs later this week.
If you need it soon, check the branch.

the include folder defined as  `BTpp_INCLUDE_DIRS` and the library ad `BTpp_LIBRARIES`

Thanks again for reporting this.

**Comment by miccol** [comment]
Solved in PR [#14](https://github.com/miccol/Behavior-Tree/pull/14)

Tested in Travis.

Let me know if there are further errors.
